evm-bytecode/EVMBLexer.g4

```diff
@@ -0,0 +1,658 @@
+   ;
+
+PUSH29
+   : 'PUSH29' Head Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs
+   ;
+
+PUSH30
+   : 'PUSH30' Head Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs
+   ;
+
+PUSH31
+   : 'PUSH31' Head Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs
+   ;
+
+PUSH32
+   : 'PUSH32' Head Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs Hexs
+   ;
+
+DUP1
+   : 'DUP1'
+   ;
+
+DUP2
+   : 'DUP2'
+   ;
+
+DUP3
+   : 'DUP3'
+   ;
+
+DUP4
+   : 'DUP4'
+   ;
+
+DUP5
+   : 'DUP5'
+   ;
+
+DUP6
+   : 'DUP6'
+   ;
+
+DUP7
+   : 'DUP7'
+   ;
+
+DUP8
+   : 'DUP8'
+   ;
+
+DUP9
+   : 'DUP9'
+   ;
+
+DUP10
+   : 'DUP10'
+   ;
+
+DUP11
+   : 'DUP11'
+   ;
+
+DUP12
+   : 'DUP12'
+   ;
+
+DUP13
+   : 'DUP13'
+   ;
+
+DUP14
+   : 'DUP14'
+   ;
+
+DUP15
+   : 'DUP15'
+   ;
+
+DUP16
+   : 'DUP16'
+   ;
+
+SWAP1
+   : 'SWAP1'
+   ;
+
+SWAP2
+   : 'SWAP2'
+   ;
+
+SWAP3
+   : 'SWAP3'
+   ;
+
+SWAP4
+   : 'SWAP4'
+   ;
+
+SWAP5
+   : 'SWAP5'
+   ;
+
+SWAP6
+   : 'SWAP6'
+   ;
+
+SWAP7
+   : 'SWAP7'
+   ;
+
+SWAP8
+   : 'SWAP8'
+   ;
+
+SWAP9
+   : 'SWAP9'
+   ;
+
+SWAP10
+   : 'SWAP10'
+   ;
+
+SWAP11
+   : 'SWAP11'
+   ;
+
+SWAP12
+   : 'SWAP12'
+   ;
+
+SWAP13
+   : 'SWAP13'
+   ;
+
+SWAP14
+   : 'SWAP14'
+   ;
+
+SWAP15
+   : 'SWAP15'
+   ;
+
+SWAP16
+   : 'SWAP16'
+   ;
+
+LOG0
+   : 'LOG0'
+   ;
+
+LOG1
+   : 'LOG1'
+   ;
+
+LOG2
+   : 'LOG2'
+   ;
+
+LOG3
+   : 'LOG3'
+   ;
+
+LOG4
+   : 'LOG4'
+   ;
+
+JUMPTO
+   : 'JUMPTO'
+   ;
+
+JUMPIF
+   : 'JUMPIF'
+   ;
+
+JUMPSUB
+   : 'JUMPSUB'
+   ;
+
+JUMPSUBV
+   : 'JUMPSUBV'
+   ;
+
+BEGINSUB
+   : 'BEGINSUB'
+   ;
+
+BEGINDATA
+   : 'BEGINDATA'
+   ;
+
+RETURNSUB
+   : 'RETURNSUB'
+   ;
+
+PUTLOCAL
+   : 'PUTLOCAL'
+   ;
+
+GETLOCA
+   : 'GETLOCA'
+   ;
+
+SLOADBYTES
+   : 'SLOADBYTES'
+   ;
+
+SSTOREBYTES
+   : 'SSTOREBYTES'
+   ;
+
+SSIZE
+   : 'SSIZE'
+   ;
+
+CREATE
+   : 'CREATE'
+   ;
+
+CALL
+   : 'CALL'
+   ;
+
+CALLCODE
+   : 'CALLCODE'
+   ;
+
+RETURN
+   : 'RETURN'
+   ;
+
+DELEGATECALL
+   : 'DELEGATECALL'
+   ;
+
+CALLBLACKBOX
+   : 'CALLBLACKBOX'
+   ;
+
+STATICCALL
+   : 'STATICCALL'
+   ;
+
+CREATE2
+   : 'CREATE2'
+   ;
+
+TXEXECGAS
+   : 'TXEXECGAS'
+   ;
+
+REVERT
+   : 'REVERT'
+   ;
+
+INVALID
+   : 'INVALID'
+   ;
+
+SELFDESTRUCT
+   : 'SELFDESTRUCT'
+   ;
+
+UNKNOW
+   : '\'' Hexs '\'' ' '? Unknown
+   ;
+
+fragment Unknown
+   : '(' 'Unknown Opcode' ')'
+   ;
+
+fragment Head
+   : ' ' '0' [xX]
+   ;
+
+fragment Hex
+   : [0-9a-fA-F]
+   ;
+
+fragment Hexs
+   : Hex Hex
+   ;
+
+WS
+   : [ \n\t\r] -> skip
```

Comment:
It's better to use `HIDDEN` channel since it preserves input.

```suggestion
   : [ \n\t\r] -> channel(HIDDEN)
```

Reply:
Done in 762802de87efbd06bc511032c2ecc531751ad2b5